airflow/www/views.py

```diff
@@ -1084,6 +1084,28 @@ def blocked(self):
             })
         return wwwutils.json_response(payload)
 
+    @expose('/latest_dag_runs')
+    @login_required
+    def latest_dag_runs(self):
```

Comment:
Please move this out from views. Ie. put the business logic into api and remove any database access from views.py. This makes it easier to unit test and to expose it by Rest API later.